pkgs/applications/office/appflowy/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchzip, autoPatchelfHook, makeWrapper, gtk3, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "appflowy";
+  version = "0.0.2";
+
+  src = fetchzip {
+    url = "https://github.com/AppFlowy-IO/appflowy/releases/download/${version}/AppFlowy-linux-x86.tar.gz";
+    sha256 = "1fvv4mlgf0vqcq5zh0zl2xr44saz0sm47r8whcywwrmcm0l66iv6";
+  };
+
+  nativeBuildInputs = [
+      autoPatchelfHook
+      makeWrapper
+  ];
+
+  buildInputs = [
+      gtk3
+      openssl
+  ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
```

Don't forget the hooks.
```suggestion
  installPhase = ''
    runHook preInstall
    
```